code/game/machinery/cryo.dm

```diff
@@ -233,6 +230,9 @@
 	return 1 // update UIs attached to this object
 
 /obj/machinery/atmospherics/unary/cryo_cell/attackby(obj/item/I, mob/user, params)
+	if(isrobot(user))
+		return
+

```

Comment:
You might want to move this robot check after the `if(istype(I, /obj/item/weapon/reagent_containers/glass))`, otherwise engiborg won't be able to deconstruct cryo tubes.